pkg/util/cluster/cluster.go

```diff
@@ -121,6 +125,24 @@ func New(log *logrus.Entry, environment env.Core, ci bool) (*Cluster, error) {
 		vaultsClient:                      keyvaultclient.NewVaultsClient(environment.Environment(), environment.SubscriptionID(), authorizer),
 	}
 
+	// Only peer if CI=true and cluster is PublicCloud
+	if ci {
+		if env.IsLocalDevelopmentMode() {
+			c.ciParentVnet = fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/dev-vpn-vnet", c.env.SubscriptionID(), c.env.ResourceGroup())
+		} else {
+			if environment.Environment().Name == azureclient.PublicCloud.Name {
+				c.ciParentVnet = "/subscriptions/26c7e39e-2dfa-4854-90f0-6bc88f7a0fb8/resourceGroups/v4ss-eastus/providers/Microsoft.Network/virtualNetworks/dev-vpn-vnet"
```

Comment:
can we remove this sub id?

Reply:
or parameterize it

Reply:
Good Catch! we don't need that if else anymore, that was for testing in development environment. Removed and used parameterized subscription and resource group